app/views/external_users/claims/interim_fee/_fields.html.haml

```diff
@@ -3,6 +3,7 @@
   %h2.bold-medium
     = t('.interim_fees')
   %p.form-hint.xsmall
+    = "All fees should be entered exclusive of VAT. The system will add VAT to fees for VAT registered users."
```

Comment:
❓ isn't this the same as the line below (that is using the locales)? 😕  (💭  If it's not, I would still consider defining a locale for this, avoiding static wording in the views).

Reply:
ooops.. forgot to wire it up again!